src/OTSubscriber.js

```diff
@@ -53,10 +53,13 @@ export default class OTSubscriber extends Component {
       this.setState({ subscriber: null });
       return;
     }
-
-    const container = document.createElement('div');
-    container.setAttribute('class', 'OTSubscriberContainer');
-    this.node.appendChild(container);
+    const { insertDefaultUI } = this.props.properties;
```

Comment:
I don't feel it's a good idea to destructure the property insertDefaultUI directly from the `props.properties` since it could be null.
I suggest to do the same we do for the publisher:
```
    const properties = this.props.properties || {};
    let container;
    if (properties.insertDefaultUI !== false) {
      .....
    }
```
This way you don't need to set insertDefaultUI in true by default, and the test case will pass.

Reply:
It cannot be null, because I have added a default prop, so if no value is being passed it will be true by default.